src/output.rs

```diff
@@ -10,15 +10,34 @@ use crate::{
     line_iterator::LineIterator,
 };
 
+// Set up display pager
+#[cfg(not(target_os = "windows"))]
+fn configure_pager(_: bool) {
+    pager::Pager::with_default_pager("less -R").setup();
```

Comment:
Could you wrap this in [Once](https://doc.rust-lang.org/std/sync/struct.Once.html) and add a note to the function docs, indicating that the function may be called multiple times safely? (Also, function docs should start with triple-slash.)

Reply:
Done! 😄